ui/app/src/components/RecentlyPublishedDashlet/RecentlyPublishedDashlet.tsx

```diff
@@ -177,17 +177,14 @@ export default function RecentlyPublishedDashlet() {
 
   // region Item Updates Propagation
   useEffect(() => {
-    const events = [itemsDeleted.type, itemsRejected.type, itemsApproved.type, itemsScheduled.type];
+    const events = [deleteContentEvent.type, publishEvent.type];
     const hostToHost$ = getHostToHostBus();
     const subscription = hostToHost$.pipe(filter((e) => events.includes(e.type))).subscribe(({ type, payload }) => {
       switch (type) {
-        case itemsApproved.type:
-        case itemsScheduled.type:
-        case itemsDeleted.type:
-        case itemsRejected.type: {
-          if (payload.targets.some((path) => itemsLookup[path])) {
-            fetchHistory();
-          }
+        case deleteContentEvent.type:
+        case workflowEvent.type:
+        case publishEvent.type: {
```

Comment:
- You have 3 cases but only listening for 2 events?
- The whole switch is kind of pointless as the handling is same for all listened events.

Reply:
Yeah the extra case was wrong, it only listens for those 2 events